### DISCUSSION TITLE
npm indexeddb not working showing version dependency

### DISCUSSION THREAD

**Discussion by senthil177cse** [GrapesJS/grapesjs#4320] [Q&A]
Hello @artf ...
    First of all thanks to your awsome GrapesJS.
    here my concern is to install indexeddb via npm its not installing showing something version dependency. check below screenshot...
    
    
![image](https://user-images.githubusercontent.com/50009624/167779619-8c2f7a87-913f-4761-9d9c-0b8021e9348e.png)

i am using angular platform to build builder using grapesjs...
can you please tell me why its getting error...

**Comment by artf** [comment]
Try to install it with `--legacy-peer-deps` flag

**Reply by senthil177cse** [reply]
i have run this command --legacy-peer-deps but again same issue...
